modules/hostap/Kconfig

```diff
@@ -265,6 +265,30 @@ config WIFI_NM_HOSTAPD_AP
 	bool "FullAP mode support based on Hostapd"
 	depends on !WIFI_NM_WPA_SUPPLICANT_INF_MON
 
+config WIFI_NM_HOSTAPD_CRYPTO_ENTERPRISE
+	bool "Hostapd crypto enterprise support"
+
+config EAP_SERVER_TLS
```

Comment:
Should we also add `EAP_SERVER_ALL` like we did for STA?

Reply:
Yes. of course, added  EAP_SERVER_ALL